apps/api/src/controllers/v1/extract-status.ts

```
@@ -0,0 +1,78 @@
+
+    if (!jobData.returnvalue) {
+      // if we got in the split-second where the redis is updated but the bull isn't
+      // just pretend it's still processing - MG
+      extract.status = "processing";
+    } else {
+      data = jobData.returnvalue ?? [];
+    }
+  }
+
+  return res.status(200).json({
+    success: extract.status === "failed" ? false : true,
```

Biome (1.9.4)

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)